src/views/news/NewsExplore/NewsExplore.vue

```diff
@@ -71,19 +72,58 @@
 
     private newsItems: (NewsItem|BlogItem)[] = [];
 
+    private calendarItems:{} = {};
```

Comment:
Can you add a better type here?

Reply:
Is this better? 
![image](https://user-images.githubusercontent.com/80924593/123294458-dec3bf80-d4e2-11eb-9277-5b2b5ea5df24.png)

Reply:
Yup, that looks good!